bedrock-lambda-cdk-dotnet/README.md

```diff
@@ -0,0 +1,54 @@
+# Extract text from images using Amazon Bedrock
```

Comment:
```suggestion
# Automatically extract text from uploaded images using Amazon Bedrock
```

Reply:
Hi @bfreiberg you suggested different title for Readme.md and example-pattern.json file, any specific reason?

how about this one
Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock

Reply:
There is a 75 characters limit for the title in the json file, whereas there is none for the README.

Reply:
I'm happy with your suggestion for this file "Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock"

Reply:
fixed